main/templates/main/homepage.html

```diff
@@ -1,32 +1,15 @@
-<!DOCTYPE html>
+{% extends "main/base_template.html" %}
+{% block content %}
 {% load static %}
-<html style="font-size: 16px;">
 
-<head>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta charset="utf-8">
-  <meta name="keywords" content="Welcome to, Hotails">
-  <meta name="description" content="">
-  <meta name="page_type" content="np-template-header-footer-from-plugin">
-  <title>Hotails</title>
   <link rel="stylesheet" href="{% static 'nicepage.css' %}" media="screen">
   <link rel="stylesheet" href="{% static 'Home.css' %}" media="screen">
-  <meta name="generator" content="Nicepage 4.6.5, nicepage.com">
   <link id="u-theme-google-font" rel="stylesheet"
     href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
   <link id="u-page-google-font" rel="stylesheet"
     href="https://fonts.googleapis.com/css?family=Roboto+Slab:100,200,300,400,500,600,700,800,900">
 
-  <meta name="theme-color" content="#478ac9">
-  <meta property="og:title" content="Home">
-  <meta property="og:type" content="website">
-</head>
-
-<body data-home-page="index.html" data-home-page-title="Home" class="u-body u-xl-mode">
-  <header class="u-clearfix u-header u-header" id="sec-23af"><a href="" class="u-image u-logo u-image-1"
-      data-image-width="300" data-image-height="300">
-      <img src="{% static 'images/Hotails-logo.jpg' %}" class="u-logo-image u-logo-image-1">
-    </a></header>
+<container data-home-page="index.html" data-home-page-title="Home" class="u-body u-xl-mode">
```

Comment:
never heard of a container tag. is that a thing? I know that in order to contain content in a tag with no special affiliation you'd usually use a <div> (while container is a class of bootstrap and not a tag)

Reply:
> never heard of a container tag. is that a thing? I know that in order to contain content in a tag with no special affiliation you'd usually use a div (while container is a class of bootstrap and not a tag)

I guess that "div" would work, but "container" is the same thing for having a "box of information" under the same "roof", for having the option to apply styling easier than specify the "div" you're meaning to.

Reply:
I see, good to know. :)